modules/statistics/php/NDB_Form_statistics.class.inc

```diff
@@ -2,70 +2,129 @@
 /**
  * The welcome menu
  *
- * @package main
+ * PHP Version 5
+ *
+ * @category Loris
+ * @package  Main
+ * @author   Dave MacFarlane <driusan@bic.mni.mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
  */
-class NDB_Form_statistics extends NDB_Form
+/**
+ * The welcome menu
+ *
+ * PHP Version 5
+ *
+ * @category Loris
+ * @package  Main
+ * @author   Dave MacFarlane <driusan@bic.mni.mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+class NDB_Form_Statistics extends NDB_Form
 {
     var $params = array();
-
+    /**
+     * Checking user's permission
+     *
+     * @return bool
+     */
     function _hasAccess()
     {
         $user =& User::singleton();
         return $user->hasPermission('data_entry');
     }
-
+    /**
+     * InCenter function
+     *
+     * @param string $centerID the value of centreID
+     * @param string $Centres  the value of Centres
+     *
+     * @return bool
+     */
     function _inCenter($centerID, $Centres)
     {
         foreach ($Centres as $Centre) {
-            if($Centre['NumericID'] == $centerID) {
+            if ($Centre['NumericID'] == $centerID) {
                 return true;
             }
         }
         return false;
     }
+    /**
+     * RenderStatsTable function
+     *
+     * @param string $sectionHeader     the value of sectionHeader
+     * @param string $tableHeader       the value of tableHeader
+     * @param string $subcats           the value of subcats
+     * @param string $visits            the value of visits
+     * @param string $dropdown_name     the value of dropdown_name
+     * @param string $dropdown_opt      the value of dropdown_opt
+     * @param string $dropdown_selected the value of dropdown_selected
+     * @param string $centres           the value of centres
+     * @param string $data              the value of data
+     * @param string $Subsection        the value of Subsection=''
+     * @param string $disclamer         the value of disclamer=''
+     * @param string $projectID         the value of projectID is null
+     *
+     * @return string
+     */
+    function renderStatsTable($sectionHeader,
+        $tableHeader,
+        $subcats,
+        $visits,
+        $dropdown_name,
+        $dropdown_opt,
+        $dropdown_selected,
+        $centres,
+        $data,
+        $Subsection="",
+        $disclamer='',
```

Comment:
@kongtiaowang why was the `$disclaimer` parameter moved down in the list ? on purpose ?

FIX:https://github.com/aces/Loris/pull/3018/files